### DISCUSSION TITLE
怎么使用自己编译的带后缀的qt

### DISCUSSION THREAD

**Discussion by Elite-stay** [xmake-io/xmake#5235] [Q&A]
比如Qt5Core_xxxxx

**Comment by star-hengxing** [comment]
估计你得把官方 qt rule 拷贝过来适配

**Reply by Elite-stay** [reply]
好像mkspecs/qconfig.pri里有QT_LIBINFIX，这个我怎么取到，有没有什么例子，就差这个了好像，写完我测试下建个pr吧

**Reply by Elite-stay** [reply]
我直接读文件分割每一行吧
